lib/renaissance_web/router.ex

```diff
@@ -21,6 +21,8 @@ defmodule RenaissanceWeb.Router do
     post "/register", RegisterController, :register
     get "/login", LoginController, :login
     post "/login", LoginController, :verify
+    get "/auctions/new", AuctionController, :new
+    post "/auctions/new", AuctionController, :create
```

Comment:
This is typically `post "/auctions"`:

https://hexdocs.pm/phoenix/routing.html#resources

Reply:
Resolved with https://github.com/xray/Renaissance/commit/9a52acc90840caae1ee2068273d642fd86942a52